### PR TITLE
Polskie tłumaczenie

### PR DESCRIPTION
**Pull request by Tymek**
LbLH8PwhXTNczLvyg1JZTi9UU92SacDCPN

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -313,6 +322,13 @@
                         <p/>
                         Een van de doelen van Litecoin was om een mining algoritme te leveren dat op hetzelfde moment kan draaien, op dezelfde hardware die gebruikt wordt om bitcoins te minen. Met de opkomst van gespecialiseerde ASICs voor Bitcoin, blijft Litecoin voldoen aan deze doelstellingen. Het is onwaarschijnlijk dat ASIC mining ontwikkeld zal worden voor Litecoin alvorens de valuta op grote schaal gebruikt wordt.
                     </h2> 
+					<% } else if (locale == 'pl') { %>
+                    <h1>Czym jest Lietcoin?</h1>
```
Comment (crazikPL) [inline views/index.ejs]: Typo:  s/Lietcoin/Litecoin

File: views/index.ejs [hunk 2]
```diff
@@ -313,6 +322,13 @@
                         <p/>
                         Een van de doelen van Litecoin was om een mining algoritme te leveren dat op hetzelfde moment kan draaien, op dezelfde hardware die gebruikt wordt om bitcoins te minen. Met de opkomst van gespecialiseerde ASICs voor Bitcoin, blijft Litecoin voldoen aan deze doelstellingen. Het is onwaarschijnlijk dat ASIC mining ontwikkeld zal worden voor Litecoin alvorens de valuta op grote schaal gebruikt wordt.
                     </h2> 
+					<% } else if (locale == 'pl') { %>
+                    <h1>Czym jest Lietcoin?</h1>
+                    <h2>
+                        Litecoin to internetowa waluta peer-to-peer umożliwiająca każdemu na świecie dokonywanie błyskawicznych transakcji. Litecoin zapewnia potwierdzenie transakcji w ciągu średnio 2,5 minuty. Jest oparty na tym samym protokole, co Bitcoin z tą różnicą że może być wydobywana efektywnie z użyciem domowego sprzętu. Używa złożonego pamięciowo algorytmu w celu udowodnienia pracy GPU lub CPU, posiadanego przez większość osób. Sieć Litecoin jest zapro­gra­mo­wana by wygenerować 84 miliony jednostek waluty.
```
Comment (crazikPL) [inline views/index.ejs]: punctation: s/protokole, co Bitcoin z/protokole co Bitcoin, z

File: views/index.ejs [hunk 3]
```diff
@@ -371,7 +387,9 @@
                         <% } else if(locale == 'de') { %>
                             Litecoin-Qt source code on GitHub            
                         <% } else if(locale == 'nl') { %>
-                            Litecoin-Qt broncode voor GitHub
+                            Litecoin-Qt broncode voor GitHub       
+                        <% } else if(locale == 'pl') { %>
+                            Kod źródłowy Litecoin-Qt w&nbsp;serwisie GitHub
```
Comment (crazikPL) [inline views/index.ejs:392]: why: nbsp?

File: views/index.ejs [hunk 4]
```diff
@@ -513,7 +531,18 @@
...
                                 <li>IRC FreeNode network channels <a href="http://webchat.freenode.net/?channels=litecoin" target="_blank" rel="nofollow">#litecoin</a> (voor algemene gebruikers) and <a href="http://webchat.freenode.net/?channels=litecoin-dev" target="_blank" rel="nofollow">#litecoin-dev</a> (voor ontwikkelaars).</li>
                             </ul> 
-                        </h2> 
+                        </h2>
+						<% } else if(locale == 'pl') { %>
+                        <h1>Społeczność</h1>
+                        <h2>Skorzystaj z pomocy rozwijających się społeczności Litecoin:
+                            <ul>
+                                <li><a href="http://forum.litecoin.net" target="_blank">Forum Litecoin</a></li>
+                                <li><a href="http://forum.litecoin.net" target="_blank">Forum Litecoin</a></li>
```
Comment (crazikPL) [inline views/index.ejs]: double line, one to remove.
p.s. maybe update to correct url: https://litecointalk.org?

File: views/index.ejs [hunk 5]
```diff
@@ -634,6 +663,15 @@
...
                             Broncode voor Litecoin-QT aan aanverwante projecten is te vinden op <a href="https://github.com/litecoin-project" target="_blank">GitHub</a>.
                         </h2>
+                    <% } else if(locale == 'pl') { %>
+                        <h1>Zasoby</h1>
+                        <h2>
+                            Ogólne informacje oraz listę usługodawców używających Litecoin znajdziesz na <a href="http://litecoin.info" target="_blank">Litecoin Wiki</a>.
+                            <p/>
+                            Aktualne statystyki sieci można przeglądać pod adresem <a href="http://ltc.block-explorer.com/charts" target="_blank">Litecoin Block Explorer Charts</a>.
+                            <p/>
+                            Kody źródłowe Litecoin-Qt oraz powiązanych projektów są dostępne w&nbsp;serwisie <a href="https://github.com/litecoin-project" target="_blank">GitHub</a>.
```
Comment (crazikPL) [inline views/index.ejs:672]: one more time: nbsp

File: views/index.ejs [hunk 6]
```diff
@@ -714,6 +752,10 @@
                         <h2>
                             Litecoin is een gratis software project uitgebracht onder de MIT/X11 licentie die gebruikers de toestemming geeft de software te draaien, aan te passen, te kopiëren en optioneel ook aangepaste kopieën van de software uit te geven. De software is uitgebracht op een transparante manier die toestaat de binaries en hun daarmee corresponderende codes onafhankelijk te verifiëren. 
                         </h2> 
+                    <% } else if(locale == 'pl') { %>
+                        <h1>Darmowe oprogramowanie</h1>
+                        <h2>Litecoin jest projektem całkowicie darmowym, opubli&shy;kowanym na licencji MIT/X11 która daje możliwość urucha&shy;miania, zmieniania, kopio&shy;wania i&nbsp;dystry&shy;bucji zmodyfi&shy;kowanych kopii oprogra&shy;mowania według uznania. Oprogra&shy;mowanie jest wydawane w&nbsp;przejrzysty sposób umo&shy;żli&shy;wiający niezależną wery&shy;fi&shy;kację plików i odpo&shy;wia&shy;da&shy;jącego im kodu źród&shy;ło&shy;wego.
```
Comment (crazikPL) [inline views/index.ejs:756]: html entities again

File: views/index.ejs [hunk 7]
```diff
@@ -800,6 +842,11 @@
                             <p/>
                             Bijgevolg krijgen handelaars snellere bevestigingstijden, terwijl ze nog altijd de mogelijkheid hebben om op meer bevestigingen te wachten wanneer zij duurdere producten verkopen.  
                         </h2> 
+                    <% } else if(locale == 'pl') { %>
+                        <h1>Blockchain</h1>
+                        <h2>
+							Litecoin radzi sobie z&nbsp;większą liczbą tran&shy;sa&shy;kcji niż jego konku&shy;rent - Bitcoin. Dzięki częstszemu genero&shy;waniu bloku sieć może utrzymać większą liczbę transakcji bez modyfikacji oprogramowania w przyszłości. <p/>Dzięki temu kupcy otrzymują szybsze potwierdzenia i wciąż mają możliwość oczekiwania na większą ilość potwierdzeń w przypadku większych transakcji.
```
Comment (crazikPL) [inline views/index.ejs:847]: html entities again

File: views/index.ejs [hunk 8]
```diff
@@ -889,6 +936,11 @@
                             <p/>
                             Dit biedt bescherming tegen portemonnee-stelende virussen en trojans , en zorgt daarnaast ook voor een extra veiligheidscontrole voordat je een betaling verzendt.
                          </h2>
+                    <% } else if(locale == 'pl') { %>
+                        <h1>Szyfrowanie portfela</h1>
+                        <h2>
+                            Szyfrowanie pozwala zabezpieczyć portfel by nikt nie mógł sprawdzić transakcji i stanu Twojego konta. Wymagane jest wprowadzenie hasła przed próbą wydania litecoinów. <p/>Ten mechanizm zabezpiecza portfel przed kradzieżą, wirusami i złośliwym oprogramowaniem oraz zapewnia możliwość sprawdzenia poprawności przed dokonaniem transakcji.
```
Comment (crazikPL) [inline views/index.ejs]: missing comma: s/portfel by/portfel, by/

**Comment by crazikPL** [comment]
Could you add also translation for last part?
it should be near line 1087:

 <% } else if(locale == 'pl') { %>
                Z zespołem deweloperów projektu Litecoina możesz się kontaktować pod adresem &lt;a href="mailto:contact@litecoin.org" target="_blank"&gt;contact@litecoin.org&lt;/a&gt;.

**Comment by Tymek** [comment]
I use HTML entities for hyphenation and to prevent "orphans". Content looks better if you care about typography. http://en.wikipedia.org/wiki/Widows_and_orphans
http://pl.wikipedia.org/wiki/Sierota_(DTP)
I'll fix the rest tomorrow. Thank you!

PS: closed by accident, sorry

**Pull request closed without merging**